examples/ddi_seal/utils.py

```diff
@@ -0,0 +1,182 @@
+import torch
+import numpy as np
+import random
+from torch_geometric.utils import negative_sampling, add_self_loops
+from torch_geometric.data import Data
+from tqdm import tqdm
+import scipy.sparse as ssp
+from scipy.sparse.csgraph import shortest_path
+
+
+def get_pos_neg_edges(split, split_edge, edge_index, num_nodes, percent=100):
```

Comment:
again, will this only be used in this example? I think these are common utilities that should be put in kale.utils so that other GNNs can reuse it.